### ISSUE TITLE
bug: message preprocessor typo

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#1724]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.0.2](https://github.com/open-wa/wa-automate-nodejs/compare/4.0.1...4.0.2) (2021-05-26)

- ✏️ typo: fix #1724 [`#1724`](https://github.com/open-wa/wa-automate-nodejs/issues/1724)
- Release 4.0.2 [`0bea284`](https://github.com/open-wa/wa-automate-nodejs/commit/0bea28447334436172b457853420d5cb3865eb85)
